stanford-earth/matson
# EARTH-216: Accessibility Improvements for Matson theme.

# READY FOR REVIEW/

# Summary
- Adds title and role to SVG Elements
- Fixes duplicate ID issue on branding lockup
- Removed unnecessary `<dd>` tags from footer

# Needed By (Date)
- Before the end of the sprint

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Clear caches
3. Review /pattern page source for changes
4. Review /home page source for changes
5. Enable voice over and tab through the new changes. Especially around the mailchimp form.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
